apps/roam/src/components/settings/LeftSidebarPersonalSettings.tsx

```diff
@@ -0,0 +1,551 @@
+import deleteBlock from "roamjs-components/writes/deleteBlock";
+import type { RoamBasicNode } from "roamjs-components/types";
+import NumberPanel from "roamjs-components/components/ConfigPanels/NumberPanel";
+import TextPanel from "roamjs-components/components/ConfigPanels/TextPanel";
+import { LeftSidebarPersonalSectionConfig } from "~/utils/getLeftSidebarSettings";
+import { extractRef, getSubTree } from "roamjs-components/util";
+import getTextByBlockUid from "roamjs-components/queries/getTextByBlockUid";
+import getPageUidByPageTitle from "roamjs-components/queries/getPageUidByPageTitle";
+import { DISCOURSE_CONFIG_PAGE_TITLE } from "~/utils/renderNodeConfigPage";
+import getCurrentUserDisplayName from "roamjs-components/queries/getCurrentUserDisplayName";
+import { getLeftSidebarPersonalSectionConfig } from "~/utils/getLeftSidebarSettings";
+
+const SectionItem = React.memo(
+  ({
+    section,
+    expandedChildLists,
+    convertToComplexSection,
+    setSettingsDialogSectionUid,
+    removeSection,
+    toggleChildrenList,
+    addChildToSection,
+    removeChild,
+    pageNames,
+  }: {
+    section: LeftSidebarPersonalSectionConfig;
+    expandedChildLists: Set<string>;
+    convertToComplexSection: (
+      section: LeftSidebarPersonalSectionConfig,
+    ) => void;
+    setSettingsDialogSectionUid: (uid: string | null) => void;
+    removeSection: (section: LeftSidebarPersonalSectionConfig) => void;
+    toggleChildrenList: (uid: string) => void;
+    addChildToSection: (
+      section: LeftSidebarPersonalSectionConfig,
+      childrenUid: string,
+      childName: string,
+    ) => Promise<void>;
+    removeChild: (
+      section: LeftSidebarPersonalSectionConfig,
+      child: RoamBasicNode,
+    ) => Promise<void>;
+    pageNames: string[];
+  }) => {
+    const ref = extractRef(section.text);
+    const blockText = getTextByBlockUid(ref);
+    const originalName = blockText || section.text;
+    const alias = section.settings?.alias?.value;
+    const [childInput, setChildInput] = useState("");
+    const [childInputKey, setChildInputKey] = useState(0);
+    const isExpanded = expandedChildLists.has(section.uid);
+
+    const handleAddChild = useCallback(async () => {
+      if (childInput && section.childrenUid) {
+        await addChildToSection(section, section.childrenUid, childInput);
+        setChildInput("");
+        setChildInputKey((prev) => prev + 1);
+      }
+    }, [childInput, section, addChildToSection]);
+
+    return (
+      <div key={section.uid} className="rounded-md border p-3 hover:bg-gray-50">
+        <div className="flex items-end justify-between">
+          <div className="flex flex-grow gap-2">
+            {!section.sectionWithoutSettingsAndChildren && (
+              <Button
+                icon={isExpanded ? "chevron-down" : "chevron-right"}
+                minimal
+                small
+                onClick={() => toggleChildrenList(section.uid)}
```

Comment:
Let's make this toggle if anywhere in the `SectionItem` is clicked.

Reply:
Single click on title takes to the page

Reply:
https://www.loom.com/share/2216bb268dac48f5bb9e7bbe5a643c84

Reply:
oh I see